cmd/api/api.go

```diff
@@ -32,6 +35,13 @@ func newClient(params paramscmd.API, opts ...api.Option) (*api.Client, error) {
 	opts = append(opts, api.WithTimeout(params.Timeout))
 	opts = append(opts, api.WithHostname(params.Hostname))
 
+	tz, err := timezone.Name()
+	if err != nil {
+		log.Debugf("failed to set up timezone option on api client: %s", err)
```

Comment:
it sounds like an error, how about `unable to detect local timezone`?

Reply:
changed to `failed to detect local timezone` tks :)